doc/eclipse.html

```diff
@@ -335,6 +343,52 @@ <h2 align="center">TestNG Eclipse plug-in</h2>
 </ul>
 </p>
 
+<h3><a name="eclipse-m2e-integration">10 - M2E Integration</a></h3>
+
+TestNG M2E Integration plugin was introduced since 6.9.10 as an optional plugin.
+It allows you to run your tests with System Properties or JVM settings, which are defined by maven-surefire-plugin or maven-failsafe-plugin of pom.xml, to be appended to the runtime TestNG process.
+
+Once this plugin installed, you can see a dedicate preference page (workspace level settings):
+<p align="center">
+<img border="0" src="http://testng.org/pictures/m2e-pref.png" />
+</p>
+or, project properties page (as name implied, can have project specific settings to override the workspace ones:
+<p align="center">
+<img border="0" src="http://testng.org/pictures/m2e-prop.png" />
+</p>
+
+Let's say there is maven-surefire-plugin confguration in your pom.xml:
+<pre class="brush: xml">
+        &lt;artifactId&gt;maven-surefire-plugin&lt;/artifactId&gt;
+        &lt;configuration&gt;
+          &lt;suiteXmlFiles&gt;
+            &lt;suiteXmlFile&gt;test-suite/testng.xml&lt;/suiteXmlFile&gt;
+          &lt;/suiteXmlFiles&gt;
+          &lt;argLine&gt;-javaagent:${settings.localRepository}/org/aspectj/aspectjweaver/${aspectj.version}/aspectjweaver-${aspectj.version}.jar
+                        -Xmx1024m -XX:MaxPermSize=512m -Xms256m -Xmx1024m -XX:PermSize=128m 
+                        -Dversion=${project.version}
+          &lt;/argLine&gt;
+          &lt;systemPropertyVariables&gt;
+            &lt;foo&gt;${foo.value}&lt;/foo&gt;
+          &lt;/systemPropertyVariables&gt;
+          &lt;environmentVariables&gt;
+            &lt;bar&gt;${bar.value}&lt;/bar&gt;
+          &lt;/environmentVariables&gt;
+        &lt;/configuration&gt;
+</pre>
+
+you can pass the following Maven configuraion parameters to TestNG process:
```

Comment:
You can pass the following Maven configuration parameters to the TestNG process: